haas/cli.py

```diff
@@ -247,17 +247,31 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+
+    # Check if config contains usable sleep_time
+    if cfg.has_option('network-daemon', 'sleep_time'):
```

Comment:
I added a statement checking for the network-daemon section in my last commit.  If either the section or the value is missing I go straight to using the default.